public/configurations/visualizations/aar-flow-sla-heatmap.json

```diff
@@ -29,6 +29,12 @@
             "params": {
                 "appName": "application",
                 "queryStartTime": "date_histo"
+            },
+            "dateParams": {
```

Comment:
@sayaji15 @ronakmshah 
Added "dateParams" section, here you just need to set the id of the "Visualization" that you need to display and it will create two params name as ##ID##-startTime and ##ID##-endTime where ##ID## will be replaced by the reference value.